.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+name: tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Since `3.6` is officially EOL I don't think we need to test it.

Reply:
Fine for me